lib/jsonpatch_exception.ex

```diff
@@ -1,14 +0,0 @@
-defmodule JsonpatchException do
```

Comment:
Why was it exchange against the error as struct?

Reply:
Sorry, my bad.
Restored!